addons/capetrader/capetrader.lua

```diff
@@ -0,0 +1,495 @@
+        else
+            capeHasBeenPrepared = false
+            currentCape = nil
+            pathItem = nil
+        end
+    else
+        windower.add_to_chat(123, 'You can\'t setup another cape while you are currently augmenting one.')
+    end
+end
+
+function startAugmentingCape(numberOfRepeats, firstAttempt)
+    inventory = windower.ffxi.get_items(inventoryBagNumber)
+    currentlyAugmenting = true
+    local augStatus = nil
+    local capeCountsafe = checkCapeCount()
+    if capeHasBeenPrepared and capeCountsafe then
+        augStatus = string.lower(checkAugLimits())
+    end
+
+    if firstAttempt and augStatus then
+        if checkThreadDustDyeSapCount(numberOfRepeats) and checkDistanceToNPC() and augStatus ~= 'maxed' and augStatus ~= 'notmatching' then
+            if string.lower(augStatus) ~= 'empty' then
+                firstTimeAug = false
+            else
+                firstTimeAug = true
+            end
+
+            local temp
+            if tonumber(numberOfRepeats) == 1 then
+                temp = 'time.'
+            else
+                temp = 'times.'
+            end
+
+            numberOfTimesToAugment = numberOfRepeats
+            windower.add_to_chat(466, 'Starting to augment your ' .. currentCape.en .. ' ' .. numberOfRepeats .. ' ' .. temp)
+
+            playerIndex = windower.ffxi.get_mob_by_target('me').index
+            tradeReady = false
+            buildTrade()
+        else
+            currentlyAugmenting = false
+            tradeReady = false
+            playerIndex = nil
+        end
+    elseif not firstAttempt and augStatus then
+        if augStatus and augStatus ~= 'maxed' then
+            tradeReady = false
+            buildTrade()
+        else
+            capeHasBeenPrepared = false
+            currentlyAugmenting = false
+            tradeReady = false
+            playerIndex = nil
+            pathItem = nil
+            currentCape = nil
+            windower.add_to_chat(466, 'Your cape is currently maxed in that augment path, ending the augment process now.')
+        end
+    elseif not capeHasBeenPrepared then
+        currentlyAugmenting = false
+        windower.add_to_chat(123, 'You have not yet setup your cape and augment information with the //ct prep command!')
+    else
+        currentlyAugmenting = false
+    end
+end
+
+function checkAugLimits()
+    local capeItem
+    for index, item in pairs(inventory) do
+        if index ~= 'max' and index ~= 'count' and index ~= 'enabled' then
+            if item.id == currentCape.id then
+                capeItem = item
+                break
+            end
+        end
+    end
+
+    local augmentTable
+    if extData.decode(capeItem).augments then
+        augmentTable = extData.decode(capeItem).augments
+    end
+
+    local augValue
+    if augmentTable then
+        if string.contains(string.lower(pathItem.en), 'thread') then
```

Comment:
I'm not sure if it's applicable here, but if the name should not just *contain* `thread` but actually *end* with it, you can use `string.endswith`. It has the advantage of avoiding false positives and is also more efficient.

And in the interest of avoiding false positives you can also add a space to the word you're checking for:
```lua
        if pathItem.en:lower():endswith(' thread') then
```